ee/pom.xml

```diff
@@ -83,7 +83,7 @@
         </dependency>
         <dependency>
             <groupId>org.jboss.spec.javax.annotation</groupId>
-            <artifactId>jboss-annotations-api_1.2_spec</artifactId>
+            <artifactId>jboss-annotations-api_1.3_spec</artifactId>
```

Comment:
I approved this, but what's the rational for changing these?

Reply:
we want 1.3 to be used everywhere given this is something that we are going use going forward.
only at runtime we expose different version.
Given the change set between two, it is harmless for pretty much all scenarios (but specific tests)